tests/command.lm.test.js

```diff
@@ -0,0 +1,53 @@
+const process = require('process')
+
+const test = require('ava')
+
+const callCli = require('./utils/call-cli')
+const { generateSiteName, createLiveTestSite } = require('./utils/create-live-test-site')
+const { createSiteBuilder } = require('./utils/site-builder')
+
+const siteName = generateSiteName('netlify-test-addons-')
+
+if (process.env.IS_FORK !== 'true') {
+  test.before(async (t) => {
+    const { siteId } = await createLiveTestSite(siteName)
+    const builder = createSiteBuilder({ siteName: 'site-with-addons' })
+    await builder.buildAsync()
+
+    t.context.execOptions = { cwd: builder.directory, env: { NETLIFY_SITE_ID: siteId } }
+    t.context.builder = builder
+  })
+
+  test.serial('netlify lm:info', async (t) => {
+    const cliResponse = await callCli(['lm:info'], t.context.execOptions)
+    t.true(cliResponse.includes('Checking Git version'))
+    t.true(cliResponse.includes('Checking Git LFS version'))
+    t.true(cliResponse.includes('Checking Git LFS filters'))
+    t.true(cliResponse.includes("Checking Netlify's Git Credentials version"))
+  })
+
+  test.serial('netlify lm:install', async (t) => {
+    const cliResponse = await callCli(['lm:install'], t.context.execOptions)
+    t.true(cliResponse.includes('Checking Git version'))
+    t.true(cliResponse.includes('Checking Git LFS version'))
+    t.true(cliResponse.includes('Checking Git LFS filters'))
+    t.true(cliResponse.includes("Installing Netlify's Git Credential Helper"))
+    t.true(cliResponse.includes("Configuring Git to use Netlify's Git Credential Helper"))
+  })
+
+  test.serial('netlify lm:setup', async (t) => {
+    // Skip the actual setup test as the site needs to be associated with the repo to provision a large media addon.
```

Comment:
I realized that it's hard to do the integration test with this. The workaround I can think of is to create a repo dedicated to this test. However, there is no good way to link the repo to the site with CLI currently (it needs multiple steps to do that action), so I wasn't able to put that into this.
I'd say that we can keep it as is for now, it's sadly not doing the provisioning but still testing something (actually this test did make me realize that the error message wasn't showing up well, which I was able to tweak).

Reply:
How about we mock the API call by setting up a mock server and use `NETLIFY_API_URL` to configure it?
I think `lm:setup` has only one API call.

Reply:
oh sorry 🤦  just realized that I dropped the ball here. I like the idea!! I think it's still actually valuable to have this "failure case" test, but I love the idea of mocking the API request and doing the success case test.

the mock success response will look like https://open-api.netlify.com/#operation/createServiceInstance but I don't think we're using any returned data so maybe mocking with empty json response with 201 is fine.

shall I do it, or will you do it?

Reply:
> shall I do it, or will you do it?

I can do it. I think it will be valuable for other tests we'd like to have

Reply:
thank you!!